.travis.yml

```diff
@@ -31,7 +31,8 @@ matrix:
   fast_finish: true
   allow_failures:
     - env: YTDL_TEST_SET=download
-    - env: JYTHON=true
+    - env: JYTHON=true; YTDL_TEST_SET=core
+    - env: JYTHON=true; YTDL_TEST_SET=download
```

Comment:
What's the point of this change?

Reply:
It did not do what I thought it would so I reverted it.